.github/workflows/build.yml

```diff
@@ -150,19 +150,19 @@ jobs:
             msan: "--with-memory-sanitizer"
             nBPF: ""
           - compiler: "cc"
-            os: macOS-latest
+            os: macos-latest
```

Comment:
Shouldn't it `macOS-latest`? Dunno if it is case-sensitive.

Reply:
GitHub ignores case when comparing strings: https://docs.github.com/en/actions/learn-github-actions/expressions
Nonetheless I will restore the original code; this change is useless